tests/framework/static_analysis.py

```diff
@@ -0,0 +1,601 @@
+# Copyright 2024 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
Can this be a rust binary instead of python script? Smth that takes binary and a seccomp json as inputs and tells if rules are correct?